pkg/azuredx/tokenprovider/clouds.go

```diff
@@ -0,0 +1,8 @@
+package tokenprovider
+
+// Must be in synch with azure cloud options in ConfigEditor.tsx
+const (
+	AzurePublic       = "azuremonitor"
+	AzureChina        = "chinaazuremonitor"
+	AzureUSGovernment = "govazuremonitor"
+)
```

Comment:
It is feasible..but very soon, like in a week or two this will be handled in the azure-sdk repo. I think we said last time that it will be two repos - one for go and one for typescript, so in that case we won't be able to share files like that. I think having that possibility would be nice though. Maybe we should reconsider having one repo for everything. But for now, let's keep it as it is?

Reply:
Fair point. Another option, for later, could be for the client to request the list from the backend through a resource handler? idk.

Reply:
Would be nice, but unfortunately it wouldn't work the first time you enter the config page since there's no backend created for it yet.